### PR TITLE
#71: close on dialog response

### LINKED ISSUE #71: About window doesn't close with close button
**Describe the bug**
Close button doesn't close the about tab

**To Reproduce**
1. Go to about tab (burger menu -> about)
2. Press the close button
3. See error

**Expected behavior**
The about window closes

**General information:**
 - Distribution: Arch
 - Installation method [e. g. built from source, installed from Flathub...]: Flathub
 - Version [e.g. 0.1.0]: 0.1.7
 - Device used [e. g. desktop, phone...]: qtile

**Stack trace:**
If applicable, run the application from a terminal and paste relevant log output.
No errors outputted

### PR DESCRIPTION
**Pull request by Douile**
Fix `AboutDialog` not closing on qtile by closing on dialog response.

Fixes #71

### CODE REVIEW

**Comment by xou816** [comment]
Thank you for doing this, this is really cool!! 
Would you mind removing my useless fix as well? 
Oh and if you want to add yourself to the contributors list feel free to do so!

**Comment by Douile** [comment]
idk how to add myself. Also have no idea how I failed this check now https://github.com/xou816/spot/pull/75/checks?check_run_id=1930811777

File: src/app/components/user_menu/user_menu.rs [hunk 1]
```diff
@@ -26,12 +26,10 @@ impl UserMenu {
...
             }),
         );
-        about.connect_destroy_event(
-            clone!(@weak about => @default-return gtk::Inhibit(false), move |_, _| {
-                about.hide();
-                gtk::Inhibit(true)
-            }),
-        );
+
+        about.connect_response(clone!(@weak about => @default-return (), move |_, _| {
```
Comment (xou816) [inline src/app/components/user_menu/user_menu.rs:30]: not sure the @default-return is needed here, it's already () by default!

Comment (xou816) [reply src/app/components/user_menu/user_menu.rs:30]: can just be `about.connect_response(clone!(@weak about => move |_, _| {`

**Comment by xou816** [comment]
It's in the window.ui file! :)

Just a single comment on your changes, looks good otherwise!

**Comment by xou816** [comment]
perfect! Thanks for your contribution :)

**Pull request merged**